PhysicsTools/NanoAOD/python/taus_cff.py

```diff
@@ -156,6 +155,11 @@ def _tauId8WPMask(pattern,doc):
 run2_miniAOD_80XLegacy.toModify(tauTable,
                                 variables = _variables80X
 )
+for era in [run2_nanoAOD_92X, run2_nanoAOD_94XMiniAODv2, \
+            run2_nanoAOD_94X2016, run2_nanoAOD_102Xv1, run2_nanoAOD_106Xv1]:
+    era.toModify(tauTable,
+                 variables = cms.PSet(tauTable.variables, _mvaAntiEVars, _mvaIsoVars2015Reduced, _mvaIsoVars2017v1, _mvaIsoVars2017v2)
+    )
```

Comment:
these lines are overwritten after, expect for the  run2_nanoAOD_94XMiniAODv1 as is missed in this list.
also the run2_nanoAOD_94XMiniAODv1 is the one that make things crush.

Reply:
Just to clarify,  run2_nanoAOD_94XMiniAODv1 is not in the list on purpose because it had an independent treatment before already. See my last comment in the main thread concerning explanation and solution of the problem.